rootfs/scheduler/__init__.py

```diff
@@ -667,6 +679,44 @@ def _delete_rc(self, namespace, name):
             error(resp, 'delete ReplicationController "{}" in Namespace "{}"',
                   name, namespace)
 
+    def _healthcheck(self, controller, path='/', port=80, delay=30, timeout=1):
+        # FIXME this logic ideally should live higher up
+        if controller['spec']['selector']['type'] not in ['web', 'cmd']:
```

Comment:
we should move this logic into _deploy_app so it stays in the model. the scheduler shouldn't be concerned about this :)